sdk/javascript/vectors/catbuffer.js

```diff
@@ -282,7 +419,7 @@ describe('catbuffer vectors', () => {
 		});
 
 		describe('Symbol', () => {
-			prepareTestCases('symbol').forEach(item => {
+			prepareTestCases('symbol', { excludes: ['other'] }).forEach(item => {
```

Comment:
why can't other be roundtripped ?

Reply:
hmm, that's likely bug on my side 🤔  will verify

Reply:
hmm, this is probably bug on my side 🤔 
(I forgot there's additional check, that only passes Transactions through TransacactionFactory)